packages/atomic/src/components/facets-v1/facet-search/facet-search-matches.tsx

```diff
@@ -27,25 +27,29 @@ export const FacetSearchMatches: FunctionalComponent<FacetSearchMatchesProps> =
 ) => {
   if (!props.numberOfMatches) {
     return (
-      <div
-        part="no-matches"
-        class="ellipsed p-3 bg-neutral-light text-neutral-dark text-sm"
-        innerHTML={matchesFound(
-          'no-matches-found-for',
-          props.query,
-          props.i18n
-        )}
-      ></div>
+      <div class="px-2">
+        <div
+          part="no-matches"
+          class="truncate p-3 bg-neutral-light text-neutral-dark text-sm rounded"
```

Comment:
Why nest a div inside another div that only has `px-2`? Is there a difference to just adding `px-2` to the original div?

Reply:
Because there is already a padding & a background on that element